packages/grafana-ui/src/components/DateTimePickers/TimeZonePicker/TimeZoneOption.tsx

```diff
@@ -65,7 +65,7 @@ export const WideTimeZoneOption: React.FC<PropsWithChildren<Props>> = (props, re
   );
 };
 
-export const CompactTimeZoneOption: React.FC<PropsWithChildren<Props>> = (props, ref) => {
+export const CompactTimeZoneOption: React.FC<React.PropsWithChildren<Props>> = (props, ref) => {
```

Comment:
minor nit - we already import `PropsWithChildren` from react in this file so could just leave this as `PropsWithChildren`

Reply:
> re: named function vs arrow function... think making sure all our components conform to our style guide is outside of the scope of this PR personally 🤔
Ok! I just wanted to make sure I understood the style guide, and I thought that was a good opportunity to do the change as we are touching the exact lines of code.